content/articles/oop-in-java-abstraction-and-encapsulation/index.md

```diff
@@ -236,27 +238,28 @@ Here is the output:
   Is the dog eating:? true
  
   The dog name is: Bingo
+  ```
  
 Now that we’ve got this out of the way let's talk about abstraction and encapsulation.
  
 ### Abstraction 
 Who could have imagined that abstraction will be a thing in programming? Who? But right now, you can’t talk about OOP without mentioning abstraction. It is a significant building block of OOP. So then, what is abstraction? This section will explain what abstraction is and how to use it as a Java developer.
 
-In abstraction, only the useful attributes of an object are on display or accessible. The low-level details are “hidden”. For example, while you’re typing on your computer. You’re interested in punching the keys believing that it produces results on your screen. You know nothing about the mechanism that produces the result, and I’m sure you haven’t even thought about it yet. Have you?
+In abstraction, only the useful attributes of an object are on display or accessible. The low-level details are *hidden*. For example, while you’re typing on your computer. You’re interested in punching the keys believing that it produces results on your screen. You know nothing about the mechanism that produces the result, and I’m sure you haven’t even thought about it yet. Have you?
```

Comment:
> For example, while you’re typing on your computer. You’re interested in punching the keys believing that it produces results on your screen.

change the word *while* to *imagine*. *while* sounds awkward here because it makes it sound like the period at the end of the sentence should be a comma. *while* makes it sound as though the two sentences are a part of the same idea/phrase.